spec/features/schemes_spec.rb

```diff
@@ -383,10 +384,12 @@
           end
 
           it "displays information about the first created location" do
-            expect(page).to have_content "SW1P4DF"
+            expect(page).to have_content "AA11AA"
             expect(page).to have_content "Some name"
             expect(page).to have_content "Self-contained house"
             expect(page).to have_content "None"
+            expect(page).to have_content "Local authority"
+            expect(page).to have_content "E09000033"
```

Comment:
Think we should be showing the label rather than the code in UI. i.e Westminster rather than E09000033